### DISCUSSION TITLE
How to control display of boolean value in results grid

### DISCUSSION THREAD

**Discussion by GTGeek88** [dbeaver/dbeaver#13592] [Q&A]
Is there a way to control how a boolean value is displayed in a results grid? I'd really like to see blank for false, a checkmark for true, and null for null. Does DBeaver allow for that type of control?

**Comment by LonwoLonwo** [comment]
Hello @GTGeek88 
You can change booleans view in Preferences -> Editors

![2021-08-20 13_07_26-Window](https://user-images.githubusercontent.com/45152336/130217915-de30ddb5-e2d0-44c1-9c62-daea6a00ba6d.png)

**Comment by GTGeek88** [comment]
Thanks. There's another place that seems to affect it, too, but this was the one that did the trick. I guess I drilled right past it.

**Comment by GTGeek88** [comment]
Well, I'm trying to mark your answer as the answer, but the small tip window about marking answers won't go away even when I click "Got It" and when I click Mark As Answer, it does nothing. I'm logged in. Not sure why it's not cooperating . . . oh, odd, as I was typing this it suddenly decided to respond to my clicks . . . over 60 seconds after the buttons were clicked. Gremlins.

**Reply by LonwoLonwo** [reply]
That's because I marked my message as an answer :upside_down_face:

**Reply by GTGeek88** [reply]
Only the OP can say what the answer is. And it was really some sort of weird webpage slowdown, as far as I can tell.  But thanks again for your help. What I was looking at was deeper in that tree view.
